docs/content/any/guides/roles/_index.md

```diff
@@ -61,6 +61,7 @@ add Oso Roles to a Node.js application.
 {{% ifLang not="node" %}}
 {{% ifLang not="python" %}}
 {{% ifLang not="ruby" %}}
+{{% ifLang not="rust" %}}
```

Comment:
I am not 100% sure (because this apparently compiles), but I think you might need a matching `{{% /ifLang }}` at the bottom of the file here.

Reply:
Fixed, thanks.